src/lib.rs

```diff
@@ -75,6 +83,39 @@ pub(crate) fn build_vector<Vector: VelloVector + Component>() -> SystemConfigs {
         .into_configs()
 }
 
+pub(crate) fn build_head<HeadEquipt: VelloVector + VectorBorder + Component>() -> SystemConfigs {
+    append_heads::<HeadEquipt>.into_configs()
+}
+
+#[allow(clippy::type_complexity)]
+fn append_heads<HeadEquipt: VelloVector + VectorBorder + Component>(
+    mut q_vectors: Query<
+        (&HeadEquipt, &Head, &mut VelloScene),
+        (Without<Stroke>, Or<(Changed<HeadEquipt>, Changed<Fill>)>),
+    >,
+    shapes: Res<Shapes>,
+    time: Res<Time>,
+) {
+    let time = time.elapsed_seconds();
+
+    for (vector, head, mut scene) in q_vectors.iter_mut() {
+        let translation = vector.border_translation(time);
+        let translation = kurbo::Vec2::new(translation.x, translation.y);
+        let tangent = vector.border_tangent(time);
+        let rotation = tangent.atan();
+
+        let transform = kurbo::Affine::default()
+            .with_translation(translation)
+            .then_rotate(rotation)
+            .then_scale(head.scale as f64);
```

Comment:
We can make `head.scale` as a f64 by default so we wouldn't need any conversion here haha.